lib/player.rb

```diff
@@ -0,0 +1,9 @@
+# player class
+class Player
+  attr_accessor :name, :board_piece
+  def initialize(name, board_piece, board)
+    @name = name
+    @board_piece = board_piece
+    @board = board
```

Comment:
The player is going to need to interact with the board, so we thought to initialize the player with a reference instead of been passing around the board in methods

Reply:
Great thinking, making the reference, instead of being specific if one of the keys of OOP.